docs/connector/spark/hive.rst

```diff
@@ -16,29 +16,29 @@
 `Hive`_
 ==========
```

Comment:
should we add supported Spark version? Iceberg, Hudi version for other section

Reply:
Add a brief description for version compatibility.